spring-session-core/src/main/java/org/springframework/session/config/annotation/web/http/EnableSpringHttpSession.java

```diff
@@ -40,7 +40,7 @@
  *
  *     {@literal @Bean}
  *     public MapSessionRepository sessionRepository() {
- *         return new MapSessionRepository();
+ *         return new MapSessionRepository(new HashMap<>());
```

Comment:
Please change this to use `new ConcurrentHashMap<>()`.

Reply:
thanks, I fixed that and also found similar case in `EnableSpringWebSession`